pkg/gh/github.go

```diff
@@ -86,6 +63,41 @@ func (o GitOptions) handleRateLimit(action func() (*github.Response, error)) err
 	return nil
 }
 
+func (o GitOptions) handleRateLimitList(action func(opt *github.ListOptions) (*github.Response, error)) error {
+	opt := &github.ListOptions{
+		PerPage: 30,
+	}
+
+	for {
+		resp, err := action(opt)
+
+		if resp.StatusCode == http.StatusUnauthorized {
+			return errors.Wrap(err, "failed to auth with GitHub, does your personal access token have the repo scope? https://github.com/settings/tokens/new?scopes=repo")
+		}
+
+		if err != nil {
+			if githubErr := github.CheckResponse(resp.Response); githubErr != nil {
+				isRateLimited, delay := o.checkRateLimiting(githubErr)
+				if isRateLimited {
+					o.Logger.Printf("retrying again later with %v second delay due to secondary rate limiting.", delay.Seconds())
+					time.Sleep(delay)
```

Comment:
Did you mean `Ratelimiter.Wait()` here?

Reply:
I've been using `time.Sleep(delay)` and has been working fine, happy to keep it TBH